lib/task/import/example/example_repositories.csv

```diff
@@ -1,3 +1,3 @@
+ARTEFAC,-1,Artefactual Archives,Evelyn McLellan,Suite 201 � 301 Sixth Street,(604)527-2056,info@artefactual.com,1-888-607-8022,https://www.artefactual.com/,"Artefactual Archives is a ficticious institution based on the real Artefactual Systems, an open source development company serving the cultural heritage sector based in New Westminster, BC, Canada. ","For more information on our team, see https://www.artefactual.com/team","Artefactual Archives collects records related to the creation and maintenance of its two primary open source projects, AtoM (Access to Memory) and Archivematica. ","Total Volume: Approximately 500 linear meters of textual records and other materials, including 274,500 photographs, 36 video webinar records, 92 slide decks, and other conference ephemera. Our two primary fonds are the AtoM fonds and the Archivematica fonds.  ","See the AtoM User, Administrator, and Developer Manuals, all found at https://www.accesstomemory.org/latest/docs/",9am-5pm Mondays to Fridays; Closed on weekends and statutory holidays. ,None,,We provide public reference services via the AtoM and Archivematica user forums,"No services are provided, but all materials are released under open licenses",,"Last updated by Dan Gillean, February 02, 2017.",ARTEFAC,"Description created by Dan Gillean on February 02, 2017",,en

```

Comment:
Looks like there may be a weird character in your example CSV data (" 201 � 301").